mithril.js

```diff
@@ -28,6 +28,19 @@
 		return "v0.2.5"
 	}
 
+	var div = document.createElement("div")
+	var frag = document.createDocumentFragment()
+
+	function toDOM(HTML){
+		div.innerHTML = HTML
```

Comment:
Here's some of the failing tests:

1. [Here - a raw text node that has no children](https://github.com/lhorie/mithril.js/blob/next/test/mithril.render.js#L357).
2. [Here - raw HTML should remain the same when updated+reverted in contenteditable `<div>`s](https://github.com/lhorie/mithril.js/blob/next/test/mithril.render.js#L1581-L1592)
3. [Here - doesn't escape ampersands in the middle](https://github.com/lhorie/mithril.js/blob/next/test/mithril.trust.js#L23-L27)